_posts/2019-02-22-3mdeb-at-embedded-world-2019.md

```diff
@@ -0,0 +1,75 @@
+---
+post_title: 3mdeb at Embedded World 2019
+author: Marta Szelecka
+layout: post
+published: true
+post_date: 2019-02-DD HH:MM:SS
+
+tags:
+    - EW2019
+    - demo
+    - conference
+    - RTE
+    - TPM2.0
+    - MuxPi
+    - BayTrail
+
+categories:
+    - Miscellaneous
+---
+
+## 3mdeb at Embedded World 2019
+
+![ew2019-logo](https://3mdeb.com/wp-content/uploads/2019/02/ew2019-logo.jpg)
```

Comment:
@artur-rs maybe we should change that graphics with this: https://cloud.3mdeb.com/index.php/s/94Cdq3AG8CdjqgH